elodie/filesystem.py

```diff
@@ -383,21 +383,22 @@ def process_file(self, _file, destination, media, **kwargs):
             os.utime(dest_path, (stat.st_atime, stat.st_mtime))
         else:
             compatability._copyfile(_file, dest_path)
-            self.set_utime(media)
+            self.set_utime(media, dest_path)
 
         db.add_hash(checksum, dest_path)
         db.update_hash_db()
 
         return dest_path
 
-    def set_utime(self, media):
+    def set_utime(self, media, file_path=None):
```

Comment:
This is only called from line 386. I would prefer to make this function a bit more explicit.

1. Rename `set_utime` to `set_utime_off_metadata`.
2. Change signature of `set_utime` to be `set_utime_off_metadata(self, metadata, file_path):` and make `file_path` required.